packages/node_modules/@webex/plugin-meetings/src/locus-info/selfUtils.js

```diff
@@ -153,18 +158,43 @@ SelfUtils.getSelfIdentity = (self) => {
 };
 
 /**
- * get the muted property from the self object
+ * get the "remote mute" property from the self object
  * @param {Object} self
  * @returns {Boolean}
  */
-SelfUtils.getMuted = (self) => {
+SelfUtils.getRemoteMuted = (self) => {
   if (!self || !self.controls || !self.controls.audio) {
     return null;
   }
 
   return self.controls.audio.muted;
 };
 
+SelfUtils.getRequestedToUnmute = (self) => {
+  if (!self || !self.controls || !self.controls.audio) {
+    return null;
+  }
+
+  return !!self.controls.audio.requestedToUnmute;
+};
+
+SelfUtils.getUnmuteAllowed = (self) => {
+  if (!self || !self.controls || !self.controls.audio) {
+    return null;
+  }
+
+  return !self.controls.audio.disallowUnmute;
+};
```

Comment:
```suggestion
SelfUtils.getUnmuteAllowed = (self) => !(self?.controls?.audio?.disallowUnmute ?? true);
```

Reply:
I think this would be wrong - it would cause getUnmuteAllowed() to return false if disallowUnmute was missing and we don't want that. I'd rather leave the code as is and return null when disallowUnmute is missing.

Reply:
Is someone checking for a null value or are they checking `falsey`? I was returning false instead of null since it looks like the other ending is checking `falsey`.

Reply:
But you can keep it like it is. I'm fine with that.

Reply:
I'm thinking about the case where `self.controls.audio` is defined, but `self.controls.audio.disallowUnmute` is undefined. In that case the, existing code returns `true` (and that's what we want, because if Locus doesn't send disallowUnmute we want to allow users to unmute themselves), while the suggested change would return `false`